src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -582,54 +600,54 @@ public void RemoveOrganelle()
 
         switch (Symmetry)
         {
-            case 0:
-            {
-                RemoveOrganelleAt(new Hex(q, r));
-                break;
-            }
-
-            case 1:
-            {
-                RemoveOrganelleAt(new Hex(q, r));
-
-                if (q != -1 * q || r != r + q)
+            case MicrobeSymmetry.None:
                 {
```

Comment:
These curly braces should be indented one level less:
```
        switch (Symmetry)
        {
            case MicrobeSymmetry.None:
            {
                RemoveOrganelleAt(new Hex(q, r));
                break;
            }

            case MicrobeSymmetry.XAxisSymmetry:
            {
```
and so on

Reply:
Thanks for the catch. Honestly, I had never seen a switch statement with curly braces before, so I must have glossed right over it. They aren't idiomatic in C#, so I went ahead and removed them from the switch statement.